### DISCUSSION TITLE
Custom UE4 engine builds

### DISCUSSION THREAD

**Discussion by Phonicx70** [adamrehn/ue4-docker#271] [Q&A] [answered]
Hello,

I am just starting to learn about containers with UE4, so this is still pretty new to me.

Having read through the documentation about using ue4-docker I can see that custom engine versions can be built, but the docs give an example by providing a github address and branch as the repository URL.

What if your custom engine resides on a Perforce server, does the -repo parameter take this into account?

Thanks

**Comment by slonopotamus** [comment, marked as answer]
ue4-docker doesn't have Perforce support currently. It only works with Git repositories.
